samples/avro-samples/confluent-schema-registry-integration/confluent-schema-registry-integration-consumer/src/main/java/sample/consumer/ConfluentAvroConsumerApplication.java

```diff
@@ -0,0 +1,26 @@
+package sample.consumer;
+
+import java.util.function.Consumer;
+
+import com.example.Sensor;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
```

Comment:
Another "maybe out fo scope" - but we should probably use current best practices such as `proxyBeanMethods = false` and package-protected visibllity where applicable (aka bean methods).

Reply:
Will be handled in #2376